src/Microsoft.VisualBasic.Core/tests/Microsoft/VisualBasic/Devices/ClockTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+
+namespace Microsoft.VisualBasic.Devices.Tests
+{
+    public class ClockTests
+    {
+        [Fact]
+        public void LocalTime()
+        {
+            var clock = new Clock();
+
+            var before = clock.LocalTime;
+            System.Threading.Thread.Sleep(10);
+
+            var now = DateTime.Now;
+            System.Threading.Thread.Sleep(10);
+
+            var after = clock.LocalTime;
+
+            Assert.True(before <= now);
```

Comment:
>= [](start = 32, length = 1)

Equal? We've delayed 10 ms, these shouldn't be equal.

Reply:
I tried that initially but one of these tests failed once.

---
In reply to: [277351954](https://github.com/dotnet/corefx/pull/37085#discussion_r277351954) [](ancestors = 277351954)